Signal/src/ViewControllers/SelectRecipientViewController.m

```diff
@@ -442,9 +481,17 @@ - (void)updateTableContents
         [self.phoneNumberTextField autoPinEdge:ALEdgeLeft toEdge:ALEdgeRight ofView:phoneNumberLabel withOffset:0];
         [self.phoneNumberTextField autoVCenterInSuperview];
 
+        // Example row.
+        UIView *examplePhoneNumberRow = [self createRowWithHeight:examplePhoneNumberRowHeight
+                                                      previousRow:phoneNumberRow
+                                                        superview:cell.contentView];
+        [examplePhoneNumberRow addSubview:self.examplePhoneNumberLabel];
+        [self.examplePhoneNumberLabel autoVCenterInSuperview];
+        [self.examplePhoneNumberLabel autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:20.f];
```

Comment:
nit: Where's the left constraint? 

Probably not an issue for the example numbers which will be relatively short, but I think it's good practice to fully constrain your view, and if you are going for a right alignment, do that on the label.